src/js/select2/dropdown/attachBody.js

```diff
@@ -32,6 +32,15 @@ define([
           self._resizeDropdown();
         });
       }
+
+	  function repositionOnSelectionChange() {
```

Comment:
Ah, yes - apologies.  It looks like your convention is a space indented file, so I'll modify.